app/views/mappings/index.html.erb

```diff
@@ -11,7 +17,7 @@
     <th></th>
   </thead>
   <tbody>
-  <%=  Mapping.find_each do |mapping| %>
+  <%  Mapping.find_each do |mapping| %>
```

Comment:
In Rails, for things like this its best practice to do the query in the controller and set an instance variable like `@mappings` that the view uses.

This slightly decouples the view from the model slightly has these benefits:

1. Maybe makes automated testing easier (for example, I can write controller tests and all the logic is in the controller)
2. It is easier to modify the logic e.g.  vary the `@mappings` list queried based on a query param
3. Could reuse the view code from multiple controllers

There are probably other reasons too but those are the only ones I remember on the top of my head.